.github/workflows/build.yml

```diff
@@ -0,0 +1,29 @@
+on:
```

Comment:
Don't we also want a preview when you create a PR? I feel like that might be nice and useful https://github.com/expo/expo-github-action?tab=readme-ov-file#create-previews-on-prs, but feel free to tell me that it isn't :smile:

Reply:
no